### DISCUSSION TITLE
Docker Changelogs: Get version from org.opencontainers.image.version label

### DISCUSSION THREAD

**Discussion by AndreasPetersen** [renovatebot/renovate#28099] [Suggest an Idea]
### Tell us more.

I am building a Docker image containing OpenTelemetry Collector, where I am adding some additional utilities to the image. I therefore version the image as `<opentelemetry-collector-version>_<additional-versioning>`. I then use `loose` versioning on my image, so that Renovate correctly picks up the latest version.

To the image, I add the labels:

```
LABEL org.opencontainers.image.source="https://github.com/open-telemetry/opentelemetry-collector-contrib"
LABEL org.opencontainers.image.version="<opentelemetry-collector-version>"
```

https://github.com/opencontainers/image-spec/blob/main/annotations.md

As per the Renovate [documentation](https://docs.renovatebot.com/modules/datasource/docker/), the changelog will be picked up based on the `org.opencontainers.image.source` label. However, because I version my image differently from OpenTelemetry Collector, it does not pick up the right version from the changelog.

I would like Renovate to use the `org.opencontainers.image.version` label for finding the version in the changelog, if specified.

**Comment by viceice** [comment]
won't possible as we only fetch the labels from latest image. we can't fetch from all versions as it's to expensive (one extra query per version, rate-limit)
